steps/git-tag-project-version-and-build-number/1.0.2/step.yml

```diff
@@ -0,0 +1,66 @@
+title: Git tag project version and build number
+summary: |
+  Git tag project version and build number
+description: |
+  This step will create Git tag with project version and build number
+website: https://github.com/ned1988/bitrise-step-git-tag-project-version-and-build-number
+source_code_url: https://github.com/ned1988/bitrise-step-git-tag-project-version-and-build-number
+support_url: https://github.com/ned1988/bitrise-step-git-tag-project-version-and-build-number
+published_at: 2019-07-29T21:04:22.46675+02:00
+source:
+  git: https://github.com/ned1988/bitrise-step-git-tag-project-version-and-build-number.git
+  commit: 970275456a8c04e40d36b65a2bf4b9a8f25c9fb8
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+- ios
```

Comment:
`ios` is not a valid host_os_tags. 
Maybe you wanted to add as a `project_type_tags` ?